scripts/install-rp.sh

```diff
@@ -169,6 +169,11 @@ sed -i '/backup-rpms-system/d' /etc/crontab
 /bin/sh -c 'echo "20 03 * * * root backup-rpms-system" >> /etc/crontab'
 echo "... line added."    
 
+echo "Adding line for daily backup schedule rpms-system in /etc/crontab..."
```

Comment:
"backup-server" instead of "daily backup schedule rpms-system" is more inline with the other entries of crontab

Reply:
see new commit...